app/src/main/java/org/oppia/app/HomeFragment.kt

```diff
@@ -0,0 +1,55 @@
+package org.oppia.app
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModelProviders
+import org.oppia.app.databinding.HomeFragmentBinding
+import org.oppia.app.model.UserAppHistory
+import org.oppia.domain.UserAppHistoryController
+import org.oppia.util.data.AsyncResult
+
+/** Fragment that contains an introduction to the app. */
+class HomeFragment : Fragment() {
+  private val userAppHistoryController = UserAppHistoryController()
+
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    val binding = HomeFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+    val viewModel = getUserAppHistoryViewModel()
+    val appUserHistory = getUserAppHistory()
+    viewModel.userAppHistoryLiveData = appUserHistory
+    // NB: Both the view model and lifecycle owner must be set in order to correctly bind LiveData elements to
+    // data-bound view models.
+    binding.let {
+      it.viewModel = viewModel
+      it.lifecycleOwner = this
+    }
+
+    // TODO(BenHenning): Mark that the user opened the app once it's persisted to disk.
```

Comment:
Could you file an issue for this and use the format `TODO(#issueNumber): ...`? We are moving to this in the main repo too for all todos, becauseo otherwise they get stale.

Reply:
Done: #70.